### ISSUE TITLE
Task "gulp test:auto" fails

### ISSUE THREAD

**Issue by e-oz** [Swiip/generator-gulp-angular#497]
Task "gulp test:auto" fails with error:

```/gulp/unit-tests.js:39
          .concat(_.pluck(files, 'path'))
          ^
TypeError: Cannot read property 'concat' of undefined
    at .../gulp/unit-tests.js:39:11
```
gulp-angular v0.11.0

reproduced in fresh generated project.

**Comment by Swiip** [comment]
Not yet released but done #480

**Comment by lbecraft** [comment]
is this fix released yet?

**Comment by mewdriller** [comment]
I'm not sure if #480 resolves this issue as the concat issue is coming from `unit-tests.js` rather than the two files edited in that PR (`inject.js` and `styles.js`).

**Comment by ronaldallanmojica** [comment]
Thanks! I also I applied it on my setup, and it is now working.

**Comment by timmaybrown** [comment]
so if this is pushed to master what do I do to get this apply to my project? I updated the generator now do i need to run the generator in the app root to this latest commit to apply? Thanks! I could obviously make the changes manually from the above commit, but wasn't sure if there was an automated way via npm or yo that is better for keeping my project up to date with the generator's fix.

**Comment by zckrs** [comment]
:tada: Just released v0.12.0 :tada:
Your issue should be fixed. I close but if you can retry your case with a new fresh install of generator you are welcome. Any feedback would be appreciated (positive comments too ^_^).
